src/unstract/adapters/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.4.0"
+__version__ = "0.4.1"
```

Comment:
Should this be version 0.5.0 as the changes in src/unstract/adapters/x2text/llm_whisperer/src/static/json_schema.json are considered as breaking API changes with the introduction/renaming of params?

Reply:
@gaya3-zipstack thanks for pointing this out - I think you're right, to be safer I should have bumped it as `0.5.0`. After taking a look at the fields, we do rename some params like `text-dump` to `dump-text` which might spell a breaking change for older versions that use the locally running instance of whisperer.

If we carry out this upgrade in all downstream uses, we can avoid this impact. Will be more careful next time

Reply:
@gaya3-zipstack Yes, agree.

@chandrasekharan-zipstack For now this is relatively ok. Let's proceed with `0.4.1`.